stow.yaml

```diff
@@ -0,0 +1,62 @@
+package:
+  name: stow
+  version: 2.4.1
+  epoch: 0
+  description: "GNU stow"
+  copyright:
+    - license: GPL-3.0
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - perl
+      - texinfo
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://ftp.gnu.org/gnu/stow/stow-${{package.version}}.tar.gz
```

Comment:
You might want to use `ftpmirror.gnu.org` here, as `ftp.gnu.org` is less stable.